huaweicloud/services/eg/data_source_huaweicloud_eg_event_streams.go

```diff
@@ -0,0 +1,407 @@
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID of the event stream.",
+						},
+						"name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The name of the event stream.",
+						},
+						"status": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The status of the event stream.",
+						},
+						"description": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The description of the event stream.",
+						},
+						"source": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "The event source configuration.",
+							Elem:        DataSourceEventStreamsSource(),
+						},
+						"sink": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "The event sink configuration.",
+							Elem:        DataSourceEventStreamSink(),
+						},
+						"rule_config": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "The configuration of event rules.",
+							Elem:        DataSourceEventRuleConfig(),
+						},
+						"option": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "The running configuration.",
+							Elem:        DataSourceEventOption(),
+						},
+						"created_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The creation time of the event stream.",
+						},
+						"updated_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The update time of the event stream.",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func DataSourceEventStreamsSource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The type of the event source.",
+			},
+			"source_kafka": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The configuration of kafka event source.",
+			},
+			"source_mobile_rocketmq": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The configuration of mobile rocketmq event source.",
+			},
+			"source_community_rocketmq": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The configuration of community RocketMQ event source.",
+			},
+			"source_dms_rocketmq": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The configuration of DMS RocketMQ event source.",
+			},
+		},
+	}
+}
+
+func DataSourceEventStreamSink() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The type of the event sink.",
```

Comment:
```suggestion
				Description: "The type of the event sink, in JSON format.",
```
do not forgot modified parameter description in schema when you modified document.

Reply:
done